README.md

```diff
@@ -1,2 +1,94 @@
-# senior_qa_test
+# Senior QA Test
 Technical test for Senior QA Analyst
+
+# Execution Profiles
+## Strict mode
+```
+cucumber -p strict
+```
+Failing API calls will thrown an exception automatically.
+
+## Debug mode
+```
+cucumber -p debug
+```
+Pauses the test execution when an exception is thrown.
+
+## Default mode
+```
+cucumber
+```
+Failing API calls will not throw an exception automatically. The test framework will carry on running, distributing the failed API response back to the calling method for further processing.
+
+e.g. Sending the failed response for further processing in a verification method.
+
+# Instructions
+1. Install Bundler and do a bundle install to install the required Gems for this framework:
```

Comment:
Maybe be more explicit in saying how you install bundler. 

`gem install bundler`

Also no mention in the readme regarding a min ruby version.